website/content/docs/auth/azure.mdx

```diff
@@ -229,6 +264,55 @@ server:
 AZURE_GO_SDK_LOG_LEVEL=DEBUG
 ```
 
+## Plugin Workload Identity Federation (WIF)
+
+<EnterpriseAlert product="vault" />
+
+The Azure auth engine supports the plugin WIF workflow, and has a source of identity called
```

Comment:
I notice this page uses "auth **engine**" everywhere, but we generally refer the feature as "auth **method**". Since we're editing the page, can we say "auth **engine**" for consistency?

```suggestion
The Azure auth method supports the plugin WIF workflow, and has a source of identity called
```